### PR TITLE
Add cookie policy to bpmn and human task explorers

### PR DESCRIPTION
**Pull request by milanperera**
screen shot 2018-03-10 at 1 41 16 pm

### CODE REVIEW

File: components/bpmn/bpmn-explorer-web/src/web/controller/privacyController.jag [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<%
+/*
+ * Copyright (c) 2005-2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Nirothipan) [inline components/bpmn/bpmn-explorer-web/src/web/controller/privacyController.jag]: Liscence header year

Comment (malakaganga) [inline components/bpmn/bpmn-explorer-web/src/web/controller/privacyController.jag]: Shall we update the year of licence header?

File: components/humantask/humantask-explorer-web/src/web/controller/privacyController.jag [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<%
+/*
+ * Copyright (c) 2005-2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (malakaganga) [inline components/humantask/humantask-explorer-web/src/web/controller/privacyController.jag]: Shall we update the year here as well?

File: components/humantask/humantask-explorer-web/src/web/controller/privacyController.jag [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+ *
```
Comment (sohaniwso2) [inline components/humantask/humantask-explorer-web/src/web/controller/privacyController.jag]: the year should be '2018' if this a newly added file

**Pull request merged**